src/Authenticator/AbstractAuthenticator.php

```diff
@@ -78,6 +85,32 @@ public function setIdentifier(IdentifierInterface $identifier)
         return $this;
     }
 
+    /**
+     * Get the last authentication result instance stored or null if none.
+     *
+     * @return \Authentication\Authenticator\ResultInterface|null
+     */
+    public function getLastResult()
```

Comment:
Should this method be added to the interface?

Reply:
Sorry for delay (was on vacation) Yes it should, and I just added. Please let me know if something else should change/improve.